{"name": "Создание записи нового струдника. Негативная проверка. Несуществующий ID", "status": "skipped", "statusDetails": {"message": "XFAIL\n\nAssertionError: assert 500 == 201", "trace": "@allure.story(\"Создание записи нового сотрудника\")\n    @allure.id(\"Vadim kandeev. Test_5\")\n    @pytest.mark.xfail()\n    @allure.title(\"Создание записи нового струдника. Негативная проверка. Несуществующий ID\")\n    def test_invalid_create_new_note_2():\n        company_id = 199885\n    \n        first_name = \"Dart\"\n        last_name = \"Waider\"\n        middle_name = \"Sith\"\n        phone = \"87776665544\"\n        url = \"www.StarWars.com\"\n    \n        nem_worker = driver.employee_post(company_id, first_name, last_name, middle_name, phone, url)\n>       assert nem_worker[1]== 201\nE       assert 500 == 201\n\ntest_employee_page_1.py:104: AssertionError"}, "start": 1677870013733, "stop": 1677870014248, "uuid": "0f381bea-290f-4c91-a763-4b415674e3c1", "historyId": "d04d1c3f8e353f1ba923f067f86ea1b5", "testCaseId": "b5efc8a3ddcfaee428ecaf9e46f1db8e", "fullName": "test_employee_page_1#test_invalid_create_new_note_2", "labels": [{"name": "story", "value": "Создание записи нового сотрудника"}, {"name": "as_id", "value": "Vadim kandeev. Test_5"}, {"name": "tag", "value": "@pytest.mark.xfail"}, {"name": "suite", "value": "test_employee_page_1"}, {"name": "host", "value": "VAdims-Mac-mini.local"}, {"name": "thread", "value": "8495-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_employee_page_1"}]}